include/nfc/tnep/tnep_timer.h

```diff
@@ -0,0 +1,51 @@
+#ifndef NFC_TNEP_TIMER_H__
```

Comment:
It seems that it could be an internal header (the one that is inside subsys directory). API is used only by the tnep.c implementation or do I miss something?

Reply:
tnep_timer is an internal header.

Reply:
Please move it then to the subsys directory. We would like to have as few files as possible in the include directory so that the user is not confused what the actual API is and what isn't.